HelloMinimal-CMake/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+# This is the most minimal version of a CMake file to compile an executable and
+# a library. Refer to HelloWorld-CMake for a more fleshed out example project.
+
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Please use 3.15.1 as the minimum version, the 3.15 release had a few nasty bugs for Swift.